src/ucp/wireup/select.c

```diff
@@ -923,11 +944,14 @@ static ucs_status_t ucp_wireup_add_rma_bw_lanes(ucp_ep_h ep,
         /* if needed for RMA, need also access for remote allocated memory */
         criteria.remote_md_flags = criteria.local_md_flags = 0;
     } else {
-        if (!(ucp_ep_get_context_features(ep) & UCP_FEATURE_TAG)) {
+        if (!(ucp_ep_get_context_features(ep) & UCP_FEATURE_TAG) &&
+            !(ucp_is_ep_mem_type(params))) {
             return UCS_OK;
         }
         /* if needed for RNDV, need only access for remote registered memory */
-        criteria.remote_md_flags = criteria.local_md_flags = UCT_MD_FLAG_REG;
+        if (!(ucp_is_ep_mem_type(params))) {
+            criteria.remote_md_flags = criteria.local_md_flags = UCT_MD_FLAG_REG;
```

Comment:
could add another `else if (ucp_is_mem_type(params)) {  ... }` block at line 946 and keep the "else" block as-is